
pcfs filesystems are not remote even though they can include ':' in the name.

diff -wpruN '--exclude=*.orig' a~/lib/mountlist.c a/lib/mountlist.c
--- a~/lib/mountlist.c	1970-01-01 00:00:00
+++ a/lib/mountlist.c	1970-01-01 00:00:00
@@ -228,6 +228,7 @@ me_remote (char const *fs_name, _GL_UNUS
    or Fs_name is equal to "-hosts" (used by autofs to mount remote fs).
    "VM" file systems like prl_fs or vboxsf are not considered remote here. */
 # define ME_REMOTE(Fs_name, Fs_type)            \
+    (strcmp (Fs_type, "pcfs") != 0 &&		\
     (strchr (Fs_name, ':') != NULL              \
      || ((Fs_name)[0] == '/'                    \
          && (Fs_name)[1] == '/'                 \
@@ -243,7 +244,7 @@ me_remote (char const *fs_name, _GL_UNUS
      || strcmp (Fs_type, "ibrix") == 0          \
      || strcmp (Fs_type, "ocfs2") == 0          \
      || strcmp (Fs_type, "vxfs") == 0           \
-     || strcmp ("-hosts", Fs_name) == 0)
+     || strcmp ("-hosts", Fs_name) == 0))
 #endif
 
 #if MOUNTED_GETMNTINFO          /* Mac OS X, FreeBSD, OpenBSD, also (obsolete) 4.4BSD */
